.github/workflows/ccruntime_e2e.yaml

```diff
@@ -25,6 +25,13 @@ jobs:
           - "az-ubuntu-2004"
           - "az-ubuntu-2204"
           - "s390x"
+          - "tdx"
+        exclude:
+          - runtimeclass: "kata-qemu"
+            instance: "tdx"
+        include:
+          - runtimeclass: "kata-qemu-tdx"
```

Comment:
I think it's because we only have an install and uninstall test at the moment, which isn't runtime class unique? I expect that when we add some e2e tests we'd be using the runtimeclass for them

Reply:
Yes, the install/uninstall tests don't use the runtimeclass but the operator.sh does check that the expected runtimeclass was installed: https://github.com/confidential-containers/operator/blob/main/tests/e2e/operator.sh#L150

Maybe that check should live in the tests for the sake of clarity?